src/components/UserFilter/UserFilter.tsx

```diff
@@ -1,72 +1,111 @@
 /* eslint-disable no-nested-ternary */
-import { FC, useMemo } from 'react';
-import { FiltersDropdownBase, FiltersDropdownItemProps, UserMenuItem } from '@taskany/bricks';
+import { useMemo } from 'react';
+import { UserPic, Tab } from '@taskany/bricks';
+import assert from 'assert';
 
-import { ActivityByIdReturnType } from '../../../trpc/inferredTypes';
 import { usePageContext } from '../../hooks/usePageContext';
+import { FilterBase } from '../FilterBase/FilterBase';
+import { FilterCheckbox } from '../FilterCheckbox';
+import { FilterTabLabel } from '../FilterTabLabel';
+import { FilterAutoCompleteInput } from '../FilterAutoCompleteInput/FilterAutoCompleteInput';
 
 import { tr } from './UserFilter.i18n';
 
-export const UserFilter: FC<{
-    users: Array<NonNullable<ActivityByIdReturnType>>;
+interface User {
+    id: string;
+    name: string | null;
+    email: string;
+    image?: string | null;
+}
+interface UserFilterProps {
+    tabName: string;
```

Comment:
I'm not sure it's connected to filter. Or in general if not using filters in other places it must be tab. Like part of FiltersPanel component.

Reply:
this component using in three cases: owners, participants and issuers with the same behavior. Also this prop is passing into the `name` prop for `Tab` component